tests/mbt/model/ccv_pss.qnt

```diff
@@ -0,0 +1,157 @@
+// This module contains logic for PSS (Partial Set Security).
+// PSS is a variant/extension of CCV that 
+// allows for only a subset of the validator set
+// to secure a consumer chain.
+// Not all logic related to PSS is inside this module, as some logic is
+// too tightly coupled with the core CCV logic,
+// which is instead found in ccv.qnt
+module ccv_pss {
+    import ccv_types.* from "./ccv"
+    import extraSpells.* from "./libraries/extraSpells"
+    import ccv_utils.* from "./ccv_utils"
+
+    // Given a base validator set, an N for a top N chain, and a set of validators that have opted in to the chain,
+    // returns the validator set that should be sent to the chain.
+    // Assumes that the value for N is valid.
+    pure def GetPSSValidatorSet(providerState: ProviderState, origValSet: ValidatorSet, consumer: Chain): ValidatorSet = {
```

Comment:
Why do we need to take the `origValSet` as a parameter here? Isn't this something we can extract from the `providerState`?